app/models/emis_redis/veteran_status.rb

```diff
@@ -78,7 +78,7 @@ def initialize(status: nil)
     # @return [Hash] A hash of veteran status properties
     #
     def validated_response
```

Comment:
Do we still need this file?

Reply:
not sure... I'll leave that for another chunk of the emis work 😄